### PR TITLE
openapi changes for #928

### PR DESCRIPTION
**Pull request by duglin**
@erikerikson wanna take a look? I don't know openapi so I'm not sure if this is correct

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: discovery/discovery.yaml [hunk 1]
```diff
@@ -131,7 +131,7 @@ components:
           schema:
             $ref: '#/components/schemas/serviceinstance'
   responses:
-    servicespostresponse:
+    servicesputresponse:
```
Comment (erikerikson) [inline discovery/discovery.yaml:134]: I think this one needs to remain `servicespostresponse`

**Review by erikerikson — CHANGES_REQUESTED** [review]
Thank you for synchronizing.  Lots of details here so well done.

There appears to be a missing `changeput` definition [between 296 and 297](https://github.com/cloudevents/spec/blob/3765d7ce87e7c92583bad2b68ae569c6a94aad51/discovery/discovery.yaml#L297).  It is referenced on [335](https://github.com/cloudevents/spec/blob/3765d7ce87e7c92583bad2b68ae569c6a94aad51/discovery/discovery.yaml#L335).  Suggested:

```
    changeput:
      type: string
      enum:
        - PUT
```

If we will be cracking the semantics of the API, it's not clear that this was the no-op presented and I think we may want to review the POST => PUT change again.

File: discovery/discovery.yaml (hunk 1 above)
Comment (duglin) [reply discovery/discovery.yaml:134]: fixed

**Comment by duglin** [comment]
Add the "changeput" - thanks for noticing

**Comment by erikerikson** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 1/27 call

**Pull request merged**